Zend/tests/stack_limit_002.phpt

```diff
@@ -0,0 +1,80 @@
+--TEST--
+Stack limit 002 - Stack limit checks with max_allowed_stack_size detection (fibers)
+--EXTENSIONS--
+zend_test
+--INI--
+fiber.stack_size=512k
```

Comment:
is this required or can the ini be unified with `Zend/tests/stack_limit_001.phpt` test?

Reply:
This is not required, it could be omitted from this particular test, but it makes the test a bit faster. Since `fiber.stack_size` has a fixed default value, omitting the setting would not test anything new/different, it would just take longer to reach the stack limit.

`Zend/tests/stack_limit_001.phpt` is a different test. It omits any ini setting to check that stack size detection works properly outside of fibers.